securitybot/bot.py

```diff
@@ -284,11 +331,10 @@ def recover_in_progress_tasks(self):
         '''
         for task in self.tasker.get_active_tasks():
             # Log new task
-            logging.info('Recovering task for {0}'.format(task.username))
-
+            task = self._store_or_update_active_task(task)
+            logging.debug('Recovering task {0}'.format(task))
```

Comment:
It was more meaningful to log the whole task with it's __dict__ while debugging, because there could be more tasks with a specific username and the associated escalation objects were not visible. But for compatibility, I added the original info logging of the username, and the task dict logging should stay as debug level imho.